client/src/main/java/com/alibaba/nacos/client/security/SecurityProxy.java

```diff
@@ -115,6 +115,7 @@ public boolean login(List<String> servers) {
                 }
             }
         } catch (Throwable ignore) {
+            SECURITY_LOGGER.error("[SecurityProxy] login  failed, servers: {}, errorMsg: {}", servers,  ignore.getMessage());
```

Comment:
The exception named ignore means the old author want to ignore this exception.

Why you want to log this exception? Can you submit an issue first?